README.md

```diff
@@ -81,8 +85,6 @@ your.token-here`.
 
 ### Linking your Discord account
 
-**Warning**: Linking your user account's token is against Discord's Terms of Service.
```

Comment:
It feels better to still leave this warning here explicitly. Or stating that linking a discord account is a self-bot which was mentioned above to be against ToS